docs/reports/review-process.md

```diff
@@ -0,0 +1,9 @@
+When you are ready to publish your report, several steps should be followed:
+
+* It must be checked by NHSE (may simply reference the corresponding paper where applicable). Contact [publications@opensafely.org](mailto:publications@opensafely.org).
+* Send your report to your co-pilot so they can review it.
+* Make sure the linked code repository is also public and has gone through the [checking process](../publishing-repo.md).
+
+
+## Next step
+[Publish your report](./publish-a-report.md)
```

Comment:
Longer term  - this should only be possible of other steps have been met?

Reply:
Correct, do you want me to call that out for this next step in particular?